planner/core/logical_plan_builder.go

```diff
@@ -312,6 +312,12 @@ func (p *LogicalJoin) extractOnCondition(conditions []expression.Expression, der
 
 func extractTableAlias(p LogicalPlan) *model.CIStr {
```

Comment:
Add some comments to tell people that if there're multiple tables, we should return nothing.

Reply:
Okay, comments already added.